### ISSUE TITLE
Module

### ISSUE THREAD

**Issue by pgatt** [hwpfeil/VCV-Rack-Additions#1]
Hi there just noticed your plugin on a GitHub crawl. Just letting you know if the only reason you are writing a module is to change the wire colours - you are able to change this in the settings if I am not mistaken. Good luck with it anyways though!

**Comment by hwpfeil** [comment]
No, not writing a module or plugin, just sharing colors for copy/paste into whatever the current wire file might be. Haven't documented this yet, the source tree changed since I uploaded this. Been working on other non-gittable projects.

**Comment by hwpfeil** [comment]
These colors were added to Rack 1.0. Difficult to hit a moving target. Version 1.0 keeps changing.
